Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Allocator.hs

```diff
@@ -47,3 +49,22 @@ data SendSearchRequestToDriverJobData = SendSearchRequestToDriverJobData
 instance JobInfoProcessor 'SendSearchRequestToDriver
 
 type instance JobContent 'SendSearchRequestToDriver = SendSearchRequestToDriverJobData
+
+data SendPaymentReminderToDriverJobData = SendPaymentReminderToDriverJobData -- combine these 2?
```

Comment:
is this comment to be removed or is it for later modifications

Reply:
I'll remove